jsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "rootDir": "./",
-    "baseUrl": "./"
+    "baseUrl": "./",
+    "jsx": "react"
```

Comment:
Can now `cmd+click` into js / jsx / components and go to definitions.

Reply:
:+1: